bin/mailing-common/mailing-list-utils.rb

```diff
@@ -176,3 +176,18 @@ def generate(name)
 
   deduped_csv
 end
+
+
+def query_all_emails_at_domain(domain)
+  puts "Emails at #{domain}"
+
+  {}.tap do |results|
+    DB[:contacts].where(Sequel.ilike(:email,"%@#{domain}")).distinct.select(:name, :email).each do |contact|
```

Comment:
The query that this generates: 
````
SELECT DISTINCT `name`, `email` FROM `contacts` WHERE (`email` LIKE '%@microsoft.com' ESCAPE '\\')
````
I think selects distinct name/email pairs, which will actually have duplicate emails. In this case it's fine because they are deduped later anyway.